.github/workflows/ci.yml

```diff
@@ -200,7 +212,7 @@ jobs:
       - name: Download stored image from artifacts
         uses: actions/download-artifact@v3.0.0
         env:
-          VAPP_IMAGE: ${{ matrix.component.Name }}_${{ github.sha }}-amd64.tar
+          VAPP_IMAGE: ${{ matrix.component.Name }}-amd64
```

Comment:
somehow the name is not fitting the one from the image-build or?

Reply:
It fits the original folder name for the amd64 build which will contain the seatadjuster.tar